roadmap.md

```diff
@@ -0,0 +1,35 @@
+Roadmap Zombienet v2
+
+## Infra
```

Comment:
### Long lived test networks deployment and management.
- deploy and manage Versi using Zombienet - similar functionality with validator manager
- obsoletes our custom solution currently built and maintained by devops